tests/routes/auth.test.js

```diff
@@ -110,4 +110,93 @@ describe('Auth Routes', () => {
       expect(typeof response.body.refreshToken).toBe('string');
     });
   });
+
+  describe('POST /signin', () => {
+    let testUser;
+
+    beforeAll(async () => {
+      testUser = await prisma.user.create({
+        data: {
+          email: 'test@example.com',
+          password: 'password',
+        },
+      });
+    });
+
+    afterAll(async () => {
+      await prisma.user.delete({
+        where: {
+          id: testUser.id,
+        },
+      });
+    });
```

Comment:
I am sorry, I am a bit confused here. Do I put these lines of code in the incorrect password test?

Reply:
<img width="1070" alt="Screenshot 2023-07-25 at 3 31 40 PM" src="https://github.com/ConnectOrlando/connectorlandoapi/assets/125603107/dbf8e192-9b81-4e53-8cbe-570fcac3a3f5">
I updated the code, but the tests keep failing. I don't know how to fix them.

Reply:
You asked to change that comment to Cannot verify user information because of security reasons. I tried updating the error messages from 400 to 401, but is still did not work.